components/Notifications.tsx

```diff
@@ -12,30 +12,35 @@ import {
   AlertDescription,
   AlertIcon,
   AlertTitle,
-  CloseButton,
   Avatar,
 } from "@chakra-ui/react";
 import React from "react";
 import GmButton from "./core/GmButton";
 import {
-  useManageSubscription,
-  useMessages,
-  useW3iAccount,
-} from "@web3inbox/widget-react";
+  useNotifications,
+  useNotificationTypes,
+  useSubscription,
+} from "@web3inbox/react";
 import Link from "next/link";
 
 import useThemeColor from "../styles/useThemeColors";
 import NotificationsIcon from "./core/NotificationsIcon";
 import SendIcon from "./core/SendIcon";
 import useSendNotification from "../utils/useSendNotification";
+import { useAccount } from "wagmi";
 
 function Notifications() {
-  const { account } = useW3iAccount();
+  const { data: notifications } = useNotifications(5, true)
```

Comment:
Can we add pagination here? `.nextPage()` or whatever

Reply:
Will do